Script/olterra.dat

```diff
@@ -182,6 +182,7 @@ decoration
     NameSingular = "pine";
     BitmapPos = 16, 320;
     HPModifier = 25;
+    LeftOverItems == pineapple { Times = 0:1; } /* an apple of a pine tree */
```

Comment:
Hm, is htere a way to make it only drop a pineapple with a very low chance?

I don't think it's crazy even now, because snow pines in Attnam give no pineapples - only (normal) pines do, and there are onyl a few in the Crystal Cave. Plus you have to know to actually chop donw the trees to get the pineapples. So right now, the player will not see enough pines to actually grant him any crazy food - instead, this is a preparatory change for the Dark Forest, where there will be limited food, but a few pines on every level...